event-svc/src/event/store.rs

```diff
@@ -44,9 +44,14 @@ impl recon::Store for EventService {
     async fn insert_many(
         &self,
         items: &[ReconItem<Self::Key>],
+        informant: &NodeId,
     ) -> ReconResult<recon::InsertResult<EventId>> {
         let res = self
-            .insert_events(items, DeliverableRequirement::Asap)
+            .insert_events(
+                items,
+                DeliverableRequirement::Asap,
+                Some(&informant.clone()),
```

Comment:
probably don't need clone since this is copy (related to other comment: should we pass owned NodeId as argument?)